akka-http-core/src/test/scala/akka/http/scaladsl/model/EntityDrainingSpec.scala

```diff
@@ -0,0 +1,95 @@
+package akka.http.scaladsl.model
+
+import java.util.concurrent.CompletableFuture
+
+import akka.Done
+import akka.actor.ActorSystem
+import akka.http.javadsl.{ model ⇒ japi }
```

Comment:
I'd rather split the java test out into Java, yes – it should be written in Java too, so we don't accidentally have JavaDSL that's not usable from Java.